Source/Editor/DualityEditor/DualityEditorApp.cs

```diff
@@ -63,6 +63,8 @@ public static class DualityEditorApp
 		private	static AutosaveFrequency			autosaveFrequency	= AutosaveFrequency.ThirtyMinutes;
 		private	static DateTime						autosaveLast		= DateTime.Now;
 		private	static string						launcherApp			= null;
+		private static string                       lastOpenScene       = null;
```

Comment:
Correct usage of spaces and tabs for alignment vs. indentation, but unfortunately the rest of the field definitions are not yet doing that, so they're not matching right now. Can you update the others too, so they match?

Reply:
My untabify in VS is not working iam afraid :(